test/framework/functional/read.go

```diff
@@ -71,6 +71,25 @@ func (f *CollectorFunctionalFramework) ReadApplicationLogsFromKafka(topic string
 	return results, err
 }
 
+func (f *CollectorFunctionalFramework) ReadApplicationLogsFromVector(consumercontainername string) (results []string, err error) {
```

Comment:
is the file format different enough we need a different function?  Is it not possible to have fluent and vector write out to the same file location and have a single function to read both?

Reply:
removed this function. thanks !!